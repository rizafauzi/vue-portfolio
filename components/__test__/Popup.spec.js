import { shallowMount } from '@vue/test-utils'
import Popup from '../Popup.vue'

describe('Popup.vue', () => {
	let wrapper = null

	beforeEach(() => {
		wrapper = shallowMount(Popup)
	})
	afterEach(() => {
		wrapper.destroy()
	})

	it('should render correctly', () => {
		
		expect(wrapper.element).toMatchSnapshot()
	})

	// it('should change text label when button clicked', () => {
	// wrapper.find('.btn-action').trigger('click.prevent');

	// expect(wrapper.vm.label).toBe('Hello');
	// })
})
