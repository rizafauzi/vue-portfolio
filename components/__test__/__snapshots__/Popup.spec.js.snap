// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popup.vue should render correctly 1`] = `
<div
  class="is-overlay"
>
  <div
    class="pop-up-wrapper"
  >
    <div
      class="close"
    >
      <svg
        fill="none"
        height="13"
        viewBox="0 0 15 15"
        width="13"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.15731 7.50008L13.796 2.86141C14.0161 2.64165 14.14 2.34343 14.1402 2.03237C14.1405 1.7213 14.0172 1.42287 13.7974 1.20272C13.5777 0.982566 13.2795 0.858733 12.9684 0.858458C12.6573 0.858183 12.3589 0.98149 12.1388 1.20125L7.50008 5.83992L2.86141 1.20125C2.64126 0.981101 2.34267 0.857422 2.03133 0.857422C1.71999 0.857422 1.4214 0.981101 1.20125 1.20125C0.981101 1.4214 0.857422 1.71999 0.857422 2.03133C0.857422 2.34267 0.981101 2.64126 1.20125 2.86141L5.83992 7.50008L1.20125 12.1388C0.981101 12.3589 0.857422 12.6575 0.857422 12.9688C0.857422 13.2802 0.981101 13.5788 1.20125 13.7989C1.4214 14.0191 1.71999 14.1427 2.03133 14.1427C2.34267 14.1427 2.64126 14.0191 2.86141 13.7989L7.50008 9.16024L12.1388 13.7989C12.3589 14.0191 12.6575 14.1427 12.9688 14.1427C13.2802 14.1427 13.5788 14.0191 13.7989 13.7989C14.0191 13.5788 14.1427 13.2802 14.1427 12.9688C14.1427 12.6575 14.0191 12.3589 13.7989 12.1388L9.15731 7.50008Z"
          fill="#070533"
        />
      </svg>
    </div>
     
    <div
      class="content"
    >
      <img
        id="image-1"
        src="~/static/background-footer.png"
      />
       
      <img
        id="image-2"
        src="~/static/background-footer.png"
      />
       
      <div
        class="tagline"
      >
        <div
          class="title"
        >
          <span>
            Thank you for subcribing!
          </span>
        </div>
      </div>
       
      <p
        class="desc"
      >
        You have successfully subscribe to our list.
        <br />
        We will keep you updated about the event.
      </p>
    </div>
  </div>
</div>
`;
